src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-focusHighlight: #1c64f2;
+$theme-primaryBackground: #1c325e;
+$theme-whiteColor: #ffffff;
+$theme-bannerFontColor: #f1416c;
+$theme-secondaryBackground: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-headingColor: #6b7280;
+$theme-inputFieldBorder: #d1d5db;
+$theme-nonActiveButtonBackground: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.88rem;
```

Comment:
why 1.88 why not 1.8? won't increasing the precision make our component brittle?

Reply:
fixed